modules/custom/az_migration/migrations/az_remote_video.yml

```
@@ -0,0 +1,78 @@
+
+id: az_remote_video
+label: Remote Video
+migration_group: az_migration
+audit: true
+migration_tags:
+  - Drupal 7
+  - Content
+  - Quickstart Content Migration
+  - Media
+
+migration_dependencies:
+  required:
+    - az_user
+    - az_files
+
+dependencies:
+  enforced:
+    module:
+      - az_migration
```

We should probably explicitly list `migrate_plus` here since that's what provides the `gate` plugin.
```suggestion
      - az_migration
      - migrate_plus
```